pkg/csr/csr.go

```diff
@@ -317,7 +318,7 @@ func matchesHostname(nodeName string, windowsInstances []*instance.Info,
 // findHostName returns the actual host name of the instance by running the 'hostname' command
 func findHostName(instanceInfo *instance.Info, instanceSigner ssh.Signer) (string, error) {
 	// We don't need to pass most args here as we just need to be able to run commands on the instance.
-	win, err := windows.New("", instanceInfo, instanceSigner)
+	win, err := windows.New("", instanceInfo, instanceSigner, v1.NonePlatformType)
```

Comment:
This can't be hardcoded to `v1.NonePlatformType`. It will cause an issue whenever findHostName is called on another platform.

If this for some reason won't cause an issue, it needs to be documented because that is not clear.